infra/app/env-template/main.tf

```
@@ -20,6 +7,12 @@ locals {
   service_name = "${local.prefix}${local.app_name}-${var.environment_name}"
 }
 
+module "aws_vpc" {
+  source       = "../../modules/vpc"
```

I bias towards naming the module something like "network" to reflect a less brand-specific meaning.

```suggestion
module "network" {
  source       = "../../modules/network"
```

or if in the future we want to support modules for different providers like Azure or GCP, we could name the module `modules/aws_network` but then we'd probably want to rename `modules/aws_service`, so maybe let's defer that to a future PR